src/components/walletconnect/WcErrorMessage/index.tsx

```diff
@@ -1,16 +1,22 @@
+import { splitError } from '@/services/walletconnect/utils'
 import { Button, Typography } from '@mui/material'
 import WcLogoHeader from '../WcLogoHeader'
 import css from './styles.module.css'
 
 const WcErrorMessage = ({ error, onClose }: { error: Error; onClose: () => void }) => {
+  const message = error.message || 'An error occurred'
+  const [summary, details] = splitError(message)
+
   return (
     <div className={css.errorContainer}>
       <WcLogoHeader error />
 
       <Typography title={error.message} className={css.errorMessage} mb={3}>
-        {error.message || 'An error occurred'}
```

Comment:
The error message layout looks a bit stretched out. Makes it seem like the details belong to the button. I think we have an `ErrorMessage` component. Is it possible to reuse it here?

<img width="479" alt="Screenshot 2023-11-01 at 09 34 44" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/7e55a6e1-8bd7-45db-9ed5-8f770b824390">

Reply:
Thank you, fixed!

<img width="538" alt="Screenshot 2023-11-01 at 10 29 33" src="https://github.com/safe-global/safe-wallet-web/assets/381895/afec08d6-b392-498c-88dd-c07dff1f8f51">